composer.json

```diff
@@ -1,8 +1,8 @@
 {
-    "name": "excelwebzone/recaptcha-bundle",
-    "description": "This bundle provides easy reCAPTCHA form field integration",
+    "name": "lethak/excelwebzonerecaptcha-bundle",
+    "description": "This bundle provides easy reCAPTCHA form field integration (forked from excelwebzone) ",
```

Comment:
actually I think this is because I wanted to use it on a project right away (and It worked well)
and shared it after it was done, so, this is not required indeed if this get merged into the official repo.